os/src/ListOps.scala

```diff
@@ -15,12 +15,12 @@ import java.nio.file.attribute.BasicFileAttributes
   * them. You can disable sorted by passing in the flag `sort = false`.
   */
 object list extends Function1[Path, IndexedSeq[Path]] {
-  def apply(src: Path, sort: Boolean = true) = {
+  def apply(src: Path, sort: Boolean = true): Array[Path] = {
```

Comment:
Shouldn't this one also return a `IndexedSeq[Path]`? The other `apply` and the inherited `Functiona1` suggests it.